docs/specification/2025-06-18/index.mdx

```diff
@@ -53,6 +53,11 @@ and tools into the ecosystem of AI applications.
 - Stateful connections
 - Server and client capability negotiation
 
+MCP deviates slightly from the JSON-RPC 2.0 specification in two respects:
+
+- Unlike base JSON-RPC, the `id` of a request/response (notifications excluded) **MUST NOT** be `null`.
```

Comment:
MCP supports notifications where `id` is set to `null` to signify that it's a fire-and-forget notification.